crypto/support/hashes/argon2/src/main/java/org/apache/shiro/crypto/support/hashes/argon2/Argon2Hash.java

```diff
@@ -209,17 +209,21 @@ public static Argon2Hash generate(
             int outputLengthBits
     ) {
         final int type;
+        Argon2Algorithm algorithm;
```

Comment:
Sorry, this refactoring brings no value and makes the code less readable

Reply:
Okay, I will revert the changes.
Sorry for the inconvenience caused.

Reply:
I am thinking of creating a new PR with small changes so it will be easier for you to review the changes and merge them into the main. Do you think I should do that?

Reply:
No. Just please reflect my comments in your current PR.
You are spamming Shiro with multiple PRs closing and opening constantly.